scripts/constants.py

```diff
@@ -143,6 +143,7 @@ class ChainId(IntEnum):
     {"contract_name": "proxy", "is_account_contract": False},
     {"contract_name": "EVM", "is_account_contract": False},
     {"contract_name": "OpenzeppelinAccount", "is_account_contract": True},
+    {"contract_name": "ERC20", "is_account_contract": False},
```

Comment:
this is meant to add ERC20.cairo as a compiled contract in our flow?

Reply:
indeed, this will just add ERC20 in the build/fixtures directory when running `make build`. It allows to have an erc20 as fee token (for example in ef-tests).